Packs/CommonScripts/Scripts/FindSimilarIncidentsV2/find_similar_incidents_test.py

```diff
@@ -118,6 +118,65 @@ def execute_command(command, args=None):
         return []
 
 
+def test_get_incidents_by_keys():
+    from FindSimilarIncidentsV2 import get_incidents_by_keys
+    res = get_incidents_by_keys({u'name': u'\U0001f489'}, 'created', '2020-10-07T19:49:37.392378+03:00', '7137', 72,
```

Comment:
Change the name to a real emoji, not the encoded one.
Like here: https://github.com/demisto/content/blob/master/Packs/Anomali_ThreatStream/Integrations/Anomali_ThreatStream_v2/Anomali_ThreatStream_v2_test.py#L81

Reply:
If there is an emoji in the incident **this** function (`get_incidents_by_keys`) accepts it that way - u'\U0001f489'